src/markdown-pages/contribute-to-quickstarts/instrument-library/index.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Instrument your Library'
+template: 'OverviewTemplate'
+description: ''
+---
+<Intro>
+If your looking to jump into building your own quickstart, the first step is instrumenting your own library to send data to New Relic. New Relic I/O gives end users the ability to slice and dice observability data around their application, server, etc. Setting up your library to get data into New Relic is critical to this process, so that the end users have a streamlined click-thorugh experience to see your data on our platform. You will need to instrument your library with at least one data type to be able to observe it via a dashboard or alert.
```

Comment:
I would like to see a more thorough explanation of _why_ the developer needs to instrument their library for the purposes of building a quickstart.

> New Relic I/O gives end users the ability to slice and dice observability data around their application, server, etc. Setting up your library to get data into New Relic is critical to this process, so that the end users have a streamlined click-thorugh experience to see your data on our platform.

To be honest, I don't really understand what this means.

> You will need to instrument your library with at least one data type to be able to observe it via a dashboard or alert.

This is more what I'm looking for, but more thorough. I'd like to see this introduction explain things like:

- Why the developer needs to instrument their library on behalf of the user
- How the developer will be able to use their instrumentation to build a quickstart
- etc.

Remember, your target audience here is an onboarding partner, and you want this to be a self-guided course. When you review your piece, it's helpful to ask yourself if you think your target audience understands what you've written enough to move on, without having read much else on the subject. Take your time with explanations. It's much better for you to overexplain than underexplain, and it's much easier to edit as well.

Reply:
I overhauled this paragraph and added some more explanation.